src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -111,6 +167,64 @@
                     "description": "Milliseconds since last update from peer"
                 }
             }
+        },
+        "diskcheck_report": {
+            "id": "diskcheck_report",
+            "description": "Result set from the redpanda diskcheck util",
+            "properties": {
+                "test_duration": {
+                    "type": "long",
+                    "description": "Length of time the test took to complete"
```

Comment:
Since this an integer, description should say what unit the value is in (presumably seconds).

Can pick this up later when adding the real checks if you like.

Reply:
Yeah this is going away, stub code for now